app/scripts/modules/core/src/reactShims/ngReact.ts

```diff
@@ -45,7 +43,6 @@ export class NgReactInjector extends ReactInject {
   public AccountRegionClusterSelector: React.ComponentClass<IAccountRegionClusterSelectorProps> = angular2react('accountRegionClusterSelectorWrapper', accountRegionClusterSelectorWrapperComponent, this.$injectorProxy) as any;
   public AddEntityTagLinks: React.ComponentClass<IAddEntityTagLinksProps>                       = angular2react('addEntityTagLinksWrapper', addEntityTagLinksWrapperComponent, this.$injectorProxy) as any;
   public ButtonBusyIndicator: React.ComponentClass<IButtonBusyIndicatorProps>                   = angular2react('buttonBusyIndicator', buttonBusyIndicatorComponent, this.$injectorProxy) as any;
-  public DiffView: React.ComponentClass<IDiffViewProps>                                         = angular2react('diffView', diffViewComponent, this.$injectorProxy) as any;
```

Comment:
This doesn't seem to be used anywhere plus there is now a fully converted React component